sdks/python/apache_beam/runners/portability/fn_api_runner/execution.py

```diff
@@ -375,6 +375,8 @@ def get_all_side_inputs():
         for o in transform.outputs.values():
           if o in s.side_inputs():
             continue
+          if o in producing_stages_by_pcoll:
```

Comment:
Could you send the runner changes as separate PR(s) ?

Reply:
Done.